test/parallel/test-http-outgoing-properties.js

```diff
@@ -51,3 +51,28 @@ const OutgoingMessage = http.OutgoingMessage;
   msg.write('asd');
   assert.strictEqual(msg.writableLength, 7);
 }
+
+{
+  const server = http.createServer(function(req, res) {
+    res.end();
+    res.on('finish', () => {
+      server.close();
```

Comment:
This can be moved outside of the `'finish'` handler since all it does is stop listening for new requests.